lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_version_helper.rb

```diff
@@ -306,15 +309,22 @@ def self.read_build_number_from_config_file(filePath)
         # Read the value of a given key from an `.xcconfig` file.
         #
         # @param [String] key The xcconfig key to get the value for
-        # @param [String] filePath The path to the `.xcconfig` file to read the value from
+        # @param [String] file_path The path to the `.xcconfig` file to read the value from
         #
         # @return [String] The value for the given key, or `nil` if the key was not found.
         #
-        def self.read_from_config_file(key, filePath)
-          File.open(filePath, 'r') do |f|
+        def self.read_from_config_file(key, file_path)
+          UI.user_error!("File #{file_path} not found") unless File.exist?(file_path)
+
+          File.open(file_path, 'r') do |f|
             f.each_line do |line|
-              line = line.strip()
-              return line.split('=')[1] if line.start_with?("#{key}=")
+              line.strip!
+              next if line.nil? || line.empty?
+
+              key_value = line.split(/\s*=\s*/)
+              if key_value[0].strip() == key
+                return key_value[1].strip()
+              end
```

Comment:
Nice improvement 👍 

What do you think about using the `xcodeproj` gem to [read the `xcconfig` file](https://rubydoc.info/gems/xcodeproj/Xcodeproj/Config) instead though? Would probably make this even more resilient, given I trust `Xcodeproj::Config` is [already well tested](https://github.com/CocoaPods/Xcodeproj/blob/master/spec/config_spec.rb#L248-L269) on those edge cases 🙂 

PS: [That `xcodeproj` gem is already a dependency of `fastlane`](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/Gemfile.lock#L175-L211) so it's not like adding it as an explicit dependency of the `release-toolkit` plugin would add any dependency or extra download to our client projects anyway, as they would already have it.

Reply:
Ah, yes, in principle it didn't occur to me, but when I read your comment in the base PR I also thought this would be much better indeed 👍

Reply:
Implemented here https://github.com/iangmaia/release-toolkit/pull/1/commits/02a99f3f822de360db6587ce97ecf19fb25b2f0e